wled00/wled.h

```diff
@@ -538,7 +538,8 @@ WLED_GLOBAL bool     serialCanTX _INIT(false);
 WLED_GLOBAL bool enableESPNow        _INIT(false);  // global on/off for ESP-NOW
 WLED_GLOBAL byte statusESPNow        _INIT(ESP_NOW_STATE_UNINIT); // state of ESP-NOW stack (0 uninitialised, 1 initialised, 2 error)
 WLED_GLOBAL bool useESPNowSync       _INIT(false);  // use ESP-NOW wireless technology for sync
-WLED_GLOBAL char linked_remote[13]   _INIT("");     // MAC of ESP-NOW remote (Wiz Mote)
+//WLED_GLOBAL char linked_remote[13]   _INIT("");     // MAC of ESP-NOW remote (Wiz Mote)
+WLED_GLOBAL std::vector<std::array<char, 13>> linked_remotes; // MAC of ESP-NOW remotes (Wiz Mote)
```

Comment:
@willmmiles can you explain why is it better to use `std::array<char,13>` instead of plain `char[13]` or what are the consquences?

IMO we don't need object of char[13] as we always know it will have exactly 12 characters and we are never going to manipulate it (except for setting content) so it is just as good to have POD.

Reply:
Found this: https://stackoverflow.com/questions/2392308/c-vector-of-char-array